src/soundanalysis.cs

```diff
@@ -153,4 +183,31 @@ interface SNResultsObserving {
 		[Export ("requestDidComplete:")]
 		void DidComplete (ISNRequest request);
 	}
+
+	[iOS (15,0), Mac (12,0), Watch (8,0), TV (15,0), MacCatalyst (15,0)]
+	enum SNClassifierIdentifier {
+		[Field ("SNClassifierIdentifierVersion1")]
+		Version1,
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SNTimeDurationConstraint /* privately conforms to NSCoding, NSCopying, and NSSecureCoding */
+	{
+		[Export ("initWithEnumeratedDurations:")]
+		IntPtr Constructor (NSValue[] enumeratedDurations);
```

Comment:
`@param enumeratedDurations A discrete set of duration values (represented as CMTime values boxed in NSValue instances) permitted by this constraint.`

Have you tried `[BindAs]` ? not sure if it handle this case...

Reply:
Yeah I tried this and showed @dalexsoto and he came to the conclusion that CMTime arrays are not supported yet